script/generate-types

```diff
@@ -8,7 +8,7 @@ cd "$(dirname "$0")/.."
 
 echo "==> Generating types..."
 
-npx openapi-typescript-codegen -i https://raw.githubusercontent.com/ministryofjustice/hmpps-approved-premises-api/main/src/main/resources/static/codegen/built-cas2-api-spec.yml -o ./server/@types/shared -c axios --exportServices false --exportCore false --useUnionTypes true
+npx openapi-typescript-codegen -i https://raw.githubusercontent.com/ministryofjustice/hmpps-approved-premises-api/cas2-326-status-updates-assessments/src/main/resources/static/codegen/built-cas2-api-spec.yml -o ./server/@types/shared -c axios --exportServices false --exportCore false --useUnionTypes true
```

Comment:
Do we want to leave this in?

Reply:
nope, will drop it once the PR is ready to merge, and the API has been done